src/fast_array_utils/stats/_sum.py

```diff
@@ -76,17 +78,20 @@ def sum_drop_keepdims(
         keepdims: bool = False,
     ) -> NDArray[Any] | types.CupyArray:
         del keepdims
-        match axis:
-            case (0 | 1 as n,):
-                axis = n
-            case (0, 1) | (1, 0):
-                axis = None
-            case tuple():  # pragma: no cover
-                msg = f"`sum` can only sum over `axis=0|1|(0,1)` but got {axis} instead"
-                raise ValueError(msg)
+        if a.ndim == 1:
```

Comment:
No check for if `axis` is something not `None`?

Reply:
Or is this somehow not important because this is an inner-computation for dask?

Reply:
yeah, we already checked that.

Reply:
fair enough